packages/website/ts/types.ts

```diff
@@ -59,25 +57,32 @@ export interface OrderToken {
     name: string;
     symbol: string;
     decimals: number;
-    address: string;
 }
 
-export interface OrderParty {
-    address: string;
-    token: OrderToken;
-    amount: string;
-    feeAmount: string;
+export interface SignedOrder {
+    maker: string;
+    taker: string;
+    makerTokenAddress: string;
+    takerTokenAddress: string;
+    makerFee: string;
+    takerFee: string;
+    makerTokenAmount: string;
+    takerTokenAmount: string;
+    expirationUnixTimestampSec: string;
+    feeRecipient: string;
+    salt: string;
+    ecSignature: ECSignature;
+    exchangeContractAddress: string;
+}
```

Comment:
This should be imported from `@0xproject/types`.

Reply:
It's impossible, cause there we have BigNumbers and here we have strings.

Reply:
Ah, I see!